cli/deployment/config.go

```diff
@@ -446,10 +446,11 @@ func newConfig() *codersdk.DeploymentConfig {
 				Default:     512,
 			},
 		},
-		Experimental: &codersdk.DeploymentConfigField[bool]{
-			Name:  "Experimental",
-			Usage: "Enable experimental features. Experimental features are not ready for production.",
-			Flag:  "experimental",
+		Experimental: &codersdk.DeploymentConfigField[codersdk.Experiments]{
+			Name:    "Experimental",
+			Usage:   "Enable one or more experiments. These are not ready for production. Separate multiple experiments with commas, or enter '*' to opt-in to all available experiments.",
+			Flag:    "experimental",
+			Default: []string{},
```

Comment:
Did you consider keeping the old flag (deprecating it) and adding a new one called `--experiments` / `--experiments-enable`. I find `--experimental feat1,feat2` to not really explain what we're doing.

Reply:
Yeah, I wouldn't mind renaming it to `--experiments=...`. We could deprecate `--experimental` but make it equivalent to `--experiments='*'`.